couchbase/transactions/pom.xml

```diff
@@ -17,6 +17,7 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-couchbase</artifactId>
+            <version>3.0.0-RC2</version>
```

Comment:
We cannot downgrade here as we will get conflicting dependencies. Users will generally see the same issue. Asking them to use a release candidate isn't an option.